src/Controller/ApplePay.php

```diff
@@ -20,15 +20,14 @@ public function __construct()
 		parent::__construct();
 
 		/** @var \WC_Settings_API  override $id */
-		$this->id = 'apple_pay';
+		$this->id                 = 'apple_pay';
 
-		/** @var \WC_Payment_Gateway overwrite for apple pay settings */
-		$this->method_title = __('payplug_apple_pay_title', 'payplug');
-		$this->method_description = "";
+		/** @var \WC_Payment_Gateway overwrite for apple pay settings  */
+		$this->method_title       = __('payplug_apple_pay_title', 'payplug');
+		$this->method_description = __('payplug_apple_pay_description', 'payplug');
 
 		$this->title = __('payplug_apple_pay_title', 'payplug');
-		$this->description = '<div id="apple-pay-button-wrapper"><apple-pay-button buttonstyle="black" type="pay" locale="'. get_locale() .'"></apple-pay-button></div>';
```

Comment:
this needs to be kept
this is just a remark since this will conflict with the current branch, so we solve it there :)

Reply:
also that, wasn't added by me, it is coming from other branch

Reply:
yes this should be kept